runners/spark/src/test/java/org/apache/beam/runners/spark/io/NumShardsTest.java

```diff
@@ -76,8 +75,7 @@ public void testText() throws Exception {
     options.setRunner(SparkRunner.class);
     Pipeline p = Pipeline.create(options);
     PCollection<String> inputWords = p.apply(Create.of(WORDS).withCoder(StringUtf8Coder.of()));
-    PCollection<String> output = inputWords.apply(new WordCount.CountWords())
-        .apply(MapElements.via(new WordCount.FormatAsTextFn()));
+    PCollection<String> output = inputWords.apply(new SimpleWordCountTest.CountWords());
```

Comment:
Yes, there are some test labeled with `NeedRunner` in `WriteTest` doing same testing. I guess we could remove this one.